pkg/minikube/problem/err_map.go

```diff
@@ -36,11 +36,43 @@ var vmProblems = map[string]match{
 		Advice: "In some environments, this message is incorrect. Try 'minikube start --no-vtx-check'",
 		Issues: []int{3900},
 	},
+	"VBOX_VERR_VMX_NO_VMX": {
+		Regexp: re(`VT-x is not available.*VERR_VMX_NO_VMX`),
+		Advice: "Please check your BIOS, and ensure that you are running without HyperV or other nested virtualization that may interfere",
+		Issues: []int{1994},
+	},
 	"VBOX_THIRD_PARTY": {
 		Regexp: re(`The virtual machine * has terminated unexpectedly during startup with exit code 1`),
 		Advice: "A third-party program may be interfering with VirtualBox. Try disabling any real-time antivirus software, reinstalling VirtualBox and rebooting.",
 		Issues: []int{3910},
 	},
+	"VBOX_BLOCKED_LOADING": {
+		Regexp: re(`NS_ERROR_FAILURE.*0x80004005`),
+		Advice: "Reinstall VirtualBox and verify that it is not blocked: System Preferences -> Security & Privacy -> General -> Some system software was blocked from loading",
+		Issues: []int{4107},
+	},
+	"VBOX_KERNEL_MODULE_NOT_LOADED": {
+		Regexp: re(`The vboxdrv kernel module is not loaded`),
+		Advice: "Run 'sudo modprobe vboxdrv' and reinstall VirtualBox if it fails.",
+		Issues: []int{4043},
+	},
+	"VBOX_DEVICE_MISSING": {
+		Regexp: re(`/dev/vboxdrv does not exist`),
+		Advice: "Run 'sudo modprobe vboxdrv' and reinstall VirtualBox if it fails.",
+		Issues: []int{3974},
+	},
+	"VBOX_HARDENING": {
```

Comment:
This is an improvement over VBOX_THIRD_PARTY above. Go ahead and remove that one.

Reply:
Good call. I've also added GOOS matching, since the advice is slightly different between Mac & Windows.